diff --git a/setup.py b/setup.py
index 75afc02..a157f9d 100755
--- a/setup.py
+++ b/setup.py
@@ -5,13 +5,17 @@ PACKAGE_NAME = 'romkan'
 
 PROJ_METADATA = '%s.json' % PROJ_NAME
 
-import os, json, imp
+import os, json, importlib.util
 
 here = os.path.abspath(os.path.dirname(__file__))
 proj_info = json.loads(open(os.path.join(here, PROJ_METADATA)).read())
 README = open(os.path.join(here, 'README.rst')).read()
 CHANGELOG = open(os.path.join(here, 'CHANGELOG.rst')).read()
-VERSION = imp.load_source('version', os.path.join(here, 'src/%s/version.py' % PACKAGE_NAME)).__version__
+
+spec = importlib.util.spec_from_file_location('version', os.path.join(here, 'src/%s/version.py' % PACKAGE_NAME))
+imp = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(imp)
+VERSION = imp.__version__
 
 from setuptools import setup, find_packages
 setup(
